### PR TITLE
Update enable-the-report-message-add-in.md

### PR DESCRIPTION
**Pull request by guruxp**
Added the necessary endpoints for the add-in to work behind a proxy.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ If you're a global administrator or an Exchange Online administrator, and Exchan
 
 - For more information on how to report a message using the Report Message feature, see [Report false positives and false negatives in Outlook](report-false-positives-and-false-negatives.md).
 
+- For organizations that have a URL filtering or security solution in place (i.e. Proxy, FireWall, etc) you will need to have ipagave.azurewebsites.net and outlook.office.com endpoints allowed to be reached on HTTPS protocol.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]:
```suggestion
- Organizations that have a URL filtering or security solution (such as Proxy and FireWall)  in place will need to have ipagave.azurewebsites.net and outlook.office.com endpoints allowed to be reached on HTTPS protocol.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @guruxp, kindly commit the suggested change. Many thanks.

**Comment by guruxp** [comment]
@JohanFreelancer9 proposed changes committed :)

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by guruxp** [comment]
@yogkumgit could you please proceed with the merge?

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Pull request merged**